<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PakTour</class>
 <widget class="QMainWindow" name="PakTour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pakistan Tour</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/SAAD COMMUNICATION/Downloads/chrome_hqmSI9koUf (2).png</normaloff>C:/Users/SAAD COMMUNICATION/Downloads/chrome_hqmSI9koUf (2).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1150</width>
      <height>689</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/SAAD COMMUNICATION/Pictures/Badshahi Mosque.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Castellar</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>Badshahi Mosque</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>270</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/SAAD COMMUNICATION/Downloads/toppng.com-right-arrow-png-transparent-800x800.png</normaloff>C:/Users/SAAD COMMUNICATION/Downloads/toppng.com-right-arrow-png-transparent-800x800.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/SAAD COMMUNICATION/Downloads/Untitled (2).png</normaloff>C:/Users/SAAD COMMUNICATION/Downloads/Untitled (2).png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>1301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity: 0.5;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/SAAD COMMUNICATION/Downloads/output-onlinepngtools.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Footlight MT Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Location: Walled City of Lahore, Punjab</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>590</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nearest City:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>550</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Available City:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>550</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lahore</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>590</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(35, 35, 35);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
